plugins/modules/identity/ipa/ipa_config.py

```diff
@@ -22,25 +29,156 @@
     description: Default e-mail domain for new users.
     aliases: ["emaildomain"]
     type: str
+  ipadefaultprimarygroup:
+    description: Default group for new users.
+    aliases: ["primarygroup"]
+    type: str
+    version_added: '2.5.0'
+  ipagroupsearchfields:
+    description: A comma-separated list of fields to search in when searching for groups.
+    aliases: ["groupsearchfields"]
+    type: str
+    version_added: '2.5.0'
+  ipahomesrootdir:
+    description: Default location of home directories.
+    aliases: ["homesrootdir"]
+    type: str
+    version_added: '2.5.0'
+  ipakrbauthzdata:
+    description: Default types of PAC supported for services
+    aliases: ["krbauthzdata"]
+    type: list
+    elements: str
+    choices: ["MS-PAC", "PAD", "nfs:NONE"]
+    version_added: '2.5.0'
+  ipamaxusernamelength:
+    description: Maximum length of usernames.
+    aliases: ["maxusernamelength"]
+    type: int
+    version_added: '2.5.0'
+  ipapwdexpadvnotify:
+    description: Notice of impending password expiration, in days.
+    aliases: ["pwdexpadvnotify"]
+    type: int
+    version_added: '2.5.0'
+  ipasearchrecordslimit:
+    description: Maximum number of records to search (-1 or 0 is unlimited).
+    aliases: ["searchrecordslimit"]
+    type: int
+    version_added: '2.5.0'
+  ipasearchtimelimit:
+    description: Maximum amount of time (seconds) for a search (-1 or 0 is unlimited).
+    aliases: ["searchtimelimit"]
+    type: int
+    version_added: '2.5.0'
+  ipauserauthtype:
+    description: The authentication type to use by default.
+    aliases: ["userauthtype"]
+    choices: ["password", "radius", "otp", "pkinit", "hardened", "disabled"]
+    type: list
+    elements: str
+    version_added: '2.5.0'
+  ipausersearchfields:
+    description: A comma-separated list of fields to search in when searching for users.
+    aliases: ["usersearchfields"]
+    type: str
```

Comment:
Same here: why not list of strings?

Reply:
Same reason as the usergroupfields, above.  Not sure why it isn't listed as multivalued in the API, but it is not.

Reply:
Resolved in d031577.